provider/blob/src/blob_data_provider.rs

```diff
@@ -90,6 +90,14 @@ pub struct BlobDataProvider {
     data: Yoke<BlobSchemaV1<'static>, Option<Cart>>,
 }
 
+impl core::fmt::Debug for BlobDataProvider {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("BlobDataProvider")
+            .field("data", &"???")
```

Comment:
```suggestion
            .field("data", &"[...]")
```

??? indicates an error